test/kommander_test.go

```diff
@@ -137,7 +140,7 @@ kubeaddonsRepository:
 		Jobs: installAutoProvisioningJobs,
 	}
 
-	addonDeployment, err := addontesters.DeployAddons(t, cluster, addons...)
+	addonDeployment, err := addontesters.DeployAddons(t, cluster, addonsWithoutCertManager...)
```

Comment:
My understanding is that cert-manager is an explicit Addon dependency, and it should be fine to:
1. Create all Addon resources (kubeaddons figures out the deploy order)
2. Wait for cert-manager Addon to be deployed
3. Install auto-provisioning helm Chart
4. Wait for all other Addons to be deployed

(3 and 4 could, in theory, happen concurrently)

Reply:
My mistake. As @mhrabovcin reminded me: the Addons (as a whole group) can't be installed, because the auto-provisioner Helm chart depends on one of the Addons.